app/views/cards/_card_form.html.erb

```diff
@@ -1,55 +1,66 @@
+      </li>
+      <li>
+        <a href="#flashcard" data-toggle="tab">Flashcard</a>
+      </li>
+      <li>
+        <a href="#organize" data-toggle="tab">Organize</a>
+      </li>
+    <% end %>
   </ul>
   <div class="tab-content">
-    
-    <!-- FLASHCARD -->
-    <div class="tab-pane" id="flashcard">
+    <!-- FILL_IN_BLANK -->
+    <div class="<%= !@card.card_type || @card.card_type === "fill_in_blank" ? "tab-pane active" : "tab-pane" %>" id="fill_in_blank">
       <%= form_for [@deck, @card] do |f| %>
         <%= render 'shared/errors', resource: @card %>
+        <%= f.hidden_field :card_type, value: 'fill_in_blank' %>
+        <div class="form-group">
+          <% f.text_area :content, class: 'form-control', rows: '5' %>
+        </div>
         <div class="form-group quotes-group">
-          <a class="btn-link" data-toggle="collapse" href="#collapseQuotesFlashcard">Guided Quotes</a>
-          <div class="collapse" id="collapseQuotesFlashcard">
-            <% @deck.quotes.each do |quote| %>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" value="">
-                  <%= quote.content %>
-                </label>
-              </div>
-            <% end %>
+          <a class="btn-link" data-toggle="collapse" href="#collapsefill_in_blank">Guiding Quotes</a>
+          <div class="collapse" id="collapsefill_in_blank">
+            <%= f.collection_radio_buttons :quote_id, @deck.quotes, :id, :content %>
           </div>
         </div>
         <div class="form-group">
-          <label>Difficulty</label>
+          <%= f.label :difficulty %>
+          <%= f.select :difficulty, Card::DIFFICULTY_LEVELS.map {|key,value| ["#{value}. #{key.titleize}",value]}, {}, { class: 'form-control'} %>
+        </div>
+        <div class="form-group">
+          <label for="card_status">Behavior *static*</label>
           <select class="form-control">
-            <option>1</option>
-            <option>2</option>
-            <option selected="selected">3</option>
+            <option selected="selected" value="random">Random</option>
+            <option value="decay">Decay</option>
           </select>
         </div>
         <div class="form-group">
-          <label>Prompt</label>
-          <textarea class="form-control" rows="5" placeholder="Flashcard prompt ..."></textarea>
+          <%= f.label :content, 'Prompt' %>
+          <%= f.text_area :content, class: 'form-control', rows: '5', placeholder: 'Fill in the blank prompt ...' %>
+          <p class="help-block">Add blanks -> #blank here#'</p>
         </div>
         <div class="form-group">
-          <label>Answer</label>
-          <textarea class="form-control" rows="4" placeholder="Flashcard answer ..."></textarea>
+          <%= f.label :status %>
+          <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
```

Comment:
You can display the `key` but you need to pass the `value` through the form. Enum requires it

`<%= f.select(:status, Card.statuses.map {|key, value| [key.titleize, value]}, {}, {:class => 'form-control'}) %>`